### DISCUSSION TITLE
How is it possible to use image size bigger than normal?

### DISCUSSION THREAD

**Discussion by leesangjoon1** [facebookresearch/detectron2#3828] [Q&A]
I heard that Swin-transformer uses image size 224x224 or 384x384.

I used Swin-L image size 384x384 

But I set the image size in .yaml file 3000x3000 (origin size is 5544 x 3692)

Then, the input image size is 3000 x 3000 or 384 x 384??

**Comment by zhanghang1989** [comment]
The train and test size can be specified in the configs https://github.com/facebookresearch/detectron2/blob/main/detectron2/config/defaults.py#L56-L65

**Reply by leesangjoon1** [reply]
> The train and test size can be specified in the configs https://github.com/facebookresearch/detectron2/blob/main/detectron2/config/defaults.py#L56-L65

Yeah I know it is specified in the configs. then, It doesn't care about model input size? (swin-transformer, resnet and so on)
